migrations/3_free_minter.js

```diff
@@ -0,0 +1,12 @@
+const Registry = artifacts.require('Registry.sol')
+const MintingController = artifacts.require('controller/MintingController.sol')
+const Resolver = artifacts.require('Resolver.sol')
+const FreeMinter = artifacts.require('util/FreeMinter.sol')
+
+module.exports = async (deployer) => {
+    const registry = await Registry.deployed();
```

Comment:
```suggestion
    const registry = await Registry.deployed()
```

A nitpick for the sake of consistency